horoscope/pyhoroscope.py

```diff
@@ -1,4 +1,13 @@
-import urllib2
+##
+##Adding support for Python3 ;-)
+##
+try:
+  import urllib2
+except ImportError:
+  import urllib.request as urllib2
+##
+
+
 from bs4 import BeautifulSoup
 import re
 
```

Comment:
Thanks for this, but I would like to stick with Python 2 for a while. Can you please undo then rebase the PR?